### PR TITLE
image_copy: Ensure bufferOffset % 4 == 0 for depth-stencil.

### PR DESCRIPTION
**Pull request by Kangz**
Issue: #

<hr>

**Requirements for PR author:**

- [X] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [X] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [X] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

File: src/webgpu/api/validation/image_copy/layout_related.spec.ts [hunk 1]
```diff
@@ -191,13 +191,17 @@ Test the computation of requiredBytesInCopy by computing the minimum data size f
...
     const copyHeight = copyHeightInBlocks * info.blockHeight;
-    const offset = offsetInBlocks * info.bytesPerBlock;
     const rowsPerImage = copyHeight + rowsPerImagePaddingInBlocks * info.blockHeight;
     const bytesPerRow =
       align(bytesInACompleteRow(copyWidth, format), bytesPerRowAlignment) +
       bytesPerRowPadding * bytesPerRowAlignment;
     const copySize = { width: copyWidth, height: copyHeight, depthOrArrayLayers: copyDepth };
 
+    let offset = offsetInBlocks * info.bytesPerBlock;
+    if (kTextureFormatInfo[format].depth || kTextureFormatInfo[format].stencil) {
```
Comment (austinEng) [inline src/webgpu/api/validation/image_copy/layout_related.spec.ts:201]:
```suggestion
    if (info.depth || info.stencil) {
```

File: src/webgpu/api/validation/image_copy/layout_related.spec.ts [hunk 2]
```diff
@@ -191,13 +191,17 @@ Test the computation of requiredBytesInCopy by computing the minimum data size f
...
+      offset = offsetInBlocks * 4;
```
Comment (austinEng) [inline src/webgpu/api/validation/image_copy/layout_related.spec.ts:202]: hm.. now the `offsetInBlocks` param doesn't make any sense for stencil8 and depth16unorm.

maybe instead in the params we should compute it based on the format inside a `.expand(...)`

**Comment by Kangz** [comment]
Done, PTAL again!

**Review by austinEng — APPROVED** [review]
Thanks for the update!

**Pull request merged**